src/OpenApi/src/Extensions/ApiDescriptionExtensions.cs

```diff
@@ -28,7 +28,7 @@ internal static class ApiDescriptionExtensions
             "HEAD" => HttpMethod.Head,
             "OPTIONS" => HttpMethod.Options,
             "TRACE" => HttpMethod.Trace,
-            "QUERY" => HttpMethod.Query,
+            "QUERY" => null, // OpenAPI as of 3.1 does not yet support HTTP QUERY
```

Comment:
```suggestion
```

Maybe we can just link to the spec here? https://spec.openapis.org/oas/v3.1.1.html#path-item-object